amd/infinityhub/cp2k/container.yaml

```diff
@@ -0,0 +1,35 @@
+docker: amdih/cp2k
```

Comment:
The docker URI here (and for the others below) should match the path of the recipe, so this should be under `amdih/cp2k`

Reply:
@vsoch thanks for the feedback, I made the change.